CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -411,10 +411,18 @@ local function CreateSystemChatMessage(settings, chattedMessage)
 
 	function this:OnResize(containerSize)
 		if this.Container and this.ChatMessage then
+
+			if InputService.VREnabled then
+				this.ChatMessage.Position = UDim2.new(0, 4, 0, 0)
+				this.ChatMessage.Size = UDim2.new(1, 0, 1, 0)
```

Comment:
Just leaving a note, don't need to fix now, but I was a little confused at first because above on line 410, we define this.chatMessage, which is assigned as the text for this.ChatMessage in CreateMessageGuiElement(). We may want to rename one of them for future clarity.